src/components/nudge/Nudge.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import PrimaryButton from '../primary-button';
+import X16 from '../../icon/fill/X16';
+
+import './Nudge.scss';
+
+export interface NudgeProps {
+    /** button text used for the primary call to action */
+    buttonText: React.ReactNode;
+    /** classname to add to the container element. */
+    className?: string;
+    /** DOM element for the content of the nudge. */
+    content: React.ReactNode;
+    /** DOM element for the illustration image */
+    illustration: React.ReactNode;
+    /** Boolean value from the parent to determine if the nudge is shown */
+    isNudgeShown: boolean;
+    /** DOM element for the header of the nudge */
+    header: React.ReactNode;
+    /** onClick callback for the nudge button */
+    onButtonClick: (event: React.SyntheticEvent<HTMLButtonElement>) => void;
+    /** onClick callback for the nudge closure button */
+    onNudgeCloseClick: (event: React.SyntheticEvent<HTMLButtonElement>) => void;
+}
+
+function Nudge({
+    buttonText,
+    className,
+    content,
+    illustration,
+    isNudgeShown,
+    header,
+    onButtonClick,
+    onNudgeCloseClick,
+}: NudgeProps) {
+    const classes = classNames(['bdl-Nudge', className], { 'bdl-is-closed': !isNudgeShown });
+
+    const closeButton = (
+        <button aria-label="close-nudge" type="button" className="bdl-Nudge-closeButton" onClick={onNudgeCloseClick}>
+            <X16 height={18} width={18} />
+        </button>
+    );
+
+    return (
+        <section className={classes}>
+            {closeButton}
+            <div className="bdl-Nudge-illustration">{illustration}</div>
+            <h1 className="bdl-Nudge-header">{header}</h1>
```

Comment:
I'm not an accessibility expert, but not sure if h1 is appropriate

Reply:
i can't see any docs that tell me otherwise (i see it a lot of examples and i don't see explicit messaging to say "NO!")